File: 백준/Bronze/4153. 직각삼각형/직각삼각형.c
#define _CRT_SECURE_NO_WARNINGS
#include <stdio.h>

int main(void){
    while(1){
        int a, b, c;
        scanf("%d %d %d",&a,&b,&c);
        if(a == 0)
            break;
        else if(a*a + b*b == c*c || b*b + c*c == a*a || c*c + a*a == b*b)
            printf("right\n");
        else
            printf("wrong\n");
    }
    return 0;
}

File: 백준/Bronze/4153. 직각삼각형/직각삼각형.py
while True:
    try:
        t = sorted(list(map(int, input().split())))
        if t[0] == t[1] == t[2] == 0:
            print()
            break
        elif t[0]**2 + t[1]**2 == t[2]**2:
            print("right")
        else:
            print("wrong")
    except:
        break